types/node-cron/index.d.ts

```diff
@@ -5,10 +5,11 @@
 //                 Richard Honor <https://github.com/RMHonor>
 //                 Ata Berk YILMAZ <https://github.com/ataberkylmz>
 //                 Alex Seidmann <https://github.com/aseidma>
+//                 Pedro Américo <https://github.com/ghostebony>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 import { EventEmitter } from 'events';
 
-export function schedule(cronExpression: string, func: ((now: Date | "manual") => void) | string, options?: ScheduleOptions): ScheduledTask;
+export function schedule(cronExpression: string, func: ((now: Date | "manual" | "init") => void) | string, options?: ScheduleOptions): ScheduledTask;
```

Comment:
Looks like this added argument can't be easily tested (compile-time).

Reply:
yeah, I know...